@xen-orchestra/web-core/lib/icons/ds-status-icon.ts

```diff
@@ -0,0 +1,253 @@
+import { defineIconPack } from '@core/packages/icon'
+import {
+  faSquare as checkboxEmpty,
+  faCircle as circleEmpty,
+  type IconDefinition,
+} from '@fortawesome/free-regular-svg-icons'
+import {
+  faBan,
+  faBolt,
+  faCheck,
+  faCircle,
+  faClose,
+  faExclamation,
+  faInfo,
+  faLock,
+  faMeteor,
+  faMinus,
+  faMoon,
+  faPause,
+  faPlay,
+  faSatelliteDish,
+  faSquare,
+  faSquareCheck,
+  faSquareMinus,
+  faStar,
+  faUserAstronaut,
+} from '@fortawesome/free-solid-svg-icons'
+
+function constructIcon(icon: IconDefinition): any {
+  return {
+    icon,
+    color: 'var(--color-neutral-txt-primary)',
+  }
+}
+
+export const dsStatusIcon = defineIconPack({
+  runnign: constructIcon(faPlay),
+
+  'runnign-circle': [
+    {
+      icon: faCircle,
+      color: 'var(--color-success-item-base)',
+    },
+    {
+      icon: faPlay,
+      color: 'var(--color-success-txt-item)',
+      translate: [0.5, 0],
+      size: 8,
+    },
+  ],
+
+  pause: constructIcon(faPause),
+
+  'pause-circle': [
+    {
+      icon: faCircle,
+      color: 'var(--color-brand-item-base)',
+    },
+    {
+      icon: faPause,
+      color: 'var(--color-brand-txt-item)',
+      size: 8,
+    },
+  ],
+
+  suspended: constructIcon(faMoon),
+
+  'suspended-circle': [
+    {
+      icon: faCircle,
+      color: 'var(--color-neutral-background-disabled)',
+    },
+    {
+      icon: faMoon,
+      color: 'var(--color-neutral-txt-secondary)',
+      translate: [-1, 0],
+      size: 13,
+    },
+  ],
+
+  halted: constructIcon(faSquare),
+
+  'halted-circle': [
+    {
+      icon: faCircle,
+      color: 'var(--color-danger-item-base)',
+    },
+    {
+      icon: faSquare,
+      color: 'var(--color-danger-txt-item)',
+      size: 7,
+    },
+  ],
+
+  'disabled-host-circle': [
```

Comment:
```suggestion
  'host-disabled-circle': [
```
To keep consistence with `sr-disabled`, `vdi-disabled` etc